client/components/LearnerViewDirective.js

```diff
@@ -52,6 +54,15 @@ tie.directive('learnerView', [function() {
                     </span>
                   </p>
                 </div>
+                <div class="tie-feedback-syntax-error">
+                <a href class="tie-feedback-syntax-error-link", ng-click="toggleSyntaxErrorHint()", 
+                        ng-show="isSyntaxErrorLinkShown">
+                  {{isSyntaxErrorShown ? 'Hide error details' : 'Display error details'}}
+                </a>
+              </div>
```

Comment:
Same comment for the lines below (make sure the tags match up and are indented properly).